.github\FUNDING.yml

```diff
@@ -1,3 +0,0 @@
-github: InventivetalentDev

```

Comment:
The filename has a backwards slash in it causing issues. There's already a FUNDING.yml in .github.

Reply:
Same with stale.yml.